websites/T/Tokopedia/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+        "name": "WilloIzCitron",
+        "id": "735410445409845251"
+    },
+    "contributors": [
+        {
+            "name": "WilloIzCitron",
+            "id": "735410445409845251"
+        }
+    ],
+    "service": "Tokopedia",
+    "description": {
+        "en": "Rich presence for Tokopedia - The Indonesian E-Commerce Site"
```

Comment:
This should be the description of Tokopedia not the description of the presence

Reply:
and it too